### ISSUE TITLE
Missing `fi` on line 27

### ISSUE THREAD

**Issue by DrRataplan** [PanderMusubi/locale-en-nl#9]
Hello there! Thanks for this work!

I just added this locale, but I ran into this:

```sh
➜  locale-en-nl git:(master) sudo ./add.sh
./add.sh: 53: Syntax error: end of file unexpected (expecting "fi")
➜  locale-en-nl git:(master) sudo bash ./add.sh
./add.sh: line 53: syntax error: unexpected end of file
```

Looking at the file, I think there is an `fi` missing around line 27. Can that be?

**Comment by mpsijm** [comment]
Looks like this was a mishap when resolving conflicts between #3 and #6, I've opened PR https://github.com/PanderMusubi/locale-en-nl/pull/10 to fix it :slightly_smiling_face:

**Comment by HanSyt** [comment]
Guess it is not solved, enclosed my modified version (added fi behind first locale-gen command.

[add.txt](https://github.com/user-attachments/files/17558627/add.txt)

**Comment by michelklaasen** [comment]
You're right. I got the same error initially and when adding fi on line 26 the script runs without problems.

However, I still get error: Bad entry 'en_NL.UTF-8' when running locale-gen. 

Does this work on Arch linux too?

**Comment by HanSyt** [comment]
I don't know about arch linux, you migth want to look at /etc/locale.gen, only have 
en_US.UTF-8 UTF-8
nl_NL.UTF-8 UTF-8
en_NL.UTF-8 UTF-8

enabled
I am running debian 12.8, raspbian (basically debian) and fedora ws40, had no problems there.

**Comment by PanderMusubi** [comment]
@michelklaasen I haven't tested it on Arch Linux. If you still have problems, plese open a new issue. Thanks
